cmake/install_python_wheel.cmake

```diff
@@ -1,3 +1,20 @@
+###############################################################################
+# Top contributors (to current version):
+#   Daniel Larraz
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2025 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# [[ Add one-line brief description here ]]
+#
+# [[ Add lengthier description here ]]
+# \todo document this file
+##
```

Comment:
@daniel-larraz can you add a comment with the description here?

Reply:
See my suggested change below.